tests/platform_tests/test_intf_fec.py

```diff
@@ -39,24 +40,15 @@ def test_verify_fec_oper_mode(duthosts, enum_rand_one_per_hwsku_frontend_hostnam
     if "broadcom" in duthost.facts.get('platform_asic'):
         pytest.skip("Skipping this test on platforms with Broadcom ASICs")
 
-    logging.info("Get output of '{}'".format("show interface status"))
-    intf_status = duthost.show_and_parse("show interface status")
+    # Get valid interfaces
+    valid_interfaces = get_valid_interfaces(duthost, SUPPORTED_SPEEDS)
 
-    for intf in intf_status:
-        sfp_presence = duthost.show_and_parse("sudo sfpshow presence -p {}"
-                                              .format(intf['interface']))
-        if sfp_presence:
-            presence = sfp_presence[0].get('presence', '').lower()
-            oper = intf.get('oper', '').lower()
-            speed = intf.get('speed', '')
-
-            if presence == "present" and oper == "up" and speed in SUPPORTED_SPEEDS:
-                # Verify the FEC operational mode is valid
-                logging.info("Get output of '{} {}'".format("show interfaces fec status", intf['interface']))
-                fec_status = duthost.show_and_parse("show interfaces fec status {}".format(intf['interface']))
-                fec = fec_status[0].get('fec oper', '').lower()
-                if fec == "n/a":
-                    pytest.fail("FEC status is N/A for interface {}".format(intf['interface']))
+    for intf in valid_interfaces:
+        # Verify the FEC operational mode is valid
+        fec = duthost.get_port_fec(intf)
```

Comment:
@vvolam The FEC operational mode is in STATE_DB whereas this API `get_port_fec()` is reading from config DB. Not sure what is the objective

Ref:-https://github.com/sonic-net/sonic-swss/blob/1847195cf02a2baa478a664a671adc95c704a810/orchagent/portsorch.cpp#L8329